enterprise/coderd/provisionerdaemons.go

```diff
@@ -221,6 +226,34 @@ func (api *API) provisionerDaemonServe(rw http.ResponseWriter, r *http.Request)
 		slog.F("tags", tags),
 	)
 
+	authCtx := ctx
+	if r.Header.Get(codersdk.ProvisionerDaemonPSK) != "" {
+		//nolint:gocritic // PSK auth means no actor in request,
```

Comment:
Maybe we should open an issue for this kind of traffic. It is a machine user, and we may highlight that in audit logs.